poseidon-permutation/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 //! An implemention of the Poseidon permutation for fixed-width
 //! hashing.
-#![cfg_attr(not(feature = "std"), no_std)]
+// #![cfg_attr(not(feature = "std"), no_std)]
+#![no_std]
```

Comment:
why? don't we want it to be opt-in?

Reply:
well with just the data structs it made sense for it to stays no_std only, now I guess it can be switched between